### ISSUE TITLE
TorchScript support

### ISSUE THREAD

**Issue by lutzroeder** [lutzroeder/netron#842]
```python
import torch
model = torch.nn.Transformer(nhead=16, num_encoder_layers=12)
src = torch.rand(10, 32, 512)
tgt = torch.rand(20, 32, 512)
trace = torch.jit.trace(model, (src, tgt))
trace.save('transformer_traced.pt')
```

**Examples:**
[alexnet.pt.zip](https://github.com/lutzroeder/netron/files/6096605/alexnet.pt.zip)
[alexnet_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096602/alexnet_traced.pt.zip)
[bert-base-uncased.pt.zip](https://github.com/lutzroeder/netron/files/7705212/bert-base-uncased.pt.zip)
[binop.pt.zip](https://github.com/user-attachments/files/22703318/binop.pt.zip)
[blitz_cifar10_tutorial.zip](https://github.com/lutzroeder/netron/files/3748500/blitz_cifar10_tutorial.zip)
[blitz_neural_networks_tutorial.zip](https://github.com/lutzroeder/netron/files/3748989/blitz_neural_networks_tutorial.zip)
[coco128-yolov8n-seg_output.torchscript.ptl.zip](https://github.com/user-attachments/files/16091260/coco128-yolov8n-seg_output.torchscript.ptl.zip)
[deeplabv3_scripted.ptl.zip](https://github.com/lutzroeder/netron/files/9562007/deeplabv3_scripted.ptl.zip)
[densenet161.pt.zip](https://github.com/lutzroeder/netron/files/6096614/densenet161.pt.zip)
[densenet161_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096613/densenet161_traced.pt.zip)
[enum_int_test.pt.zip](https://github.com/user-attachments/files/22600859/enum_int_test.pt.zip)
[fasterrcnn_resnet50_fpn.pt.zip](https://github.com/lutzroeder/netron/files/10254858/fasterrcnn_resnet50_fpn.pt.zip)
[fasterrcnn_resnet50_fpn.pt.zip](https://github.com/lutzroeder/netron/files/7677467/fasterrcnn_resnet50_fpn.pt.zip)
[gpt2.pt.zip](https://github.com/lutzroeder/netron/files/11585383/gpt2.pt.zip)
[inception_v3.pt.zip](https://github.com/lutzroeder/netron/files/6096628/inception_v3.pt.zip)
[inception_v3_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096627/inception_v3_traced.pt.zip)
[iv3_pertensor.zip](https://github.com/lutzroeder/netron/files/5212009/iv3_pertensor.zip)
[lite_real_sr.ptl.zip](https://github.com/lutzroeder/netron/files/12137599/lite_real_sr.ptl.zip)
[LMModel1.zip](https://github.com/lutzroeder/netron/files/7726055/LMModel1.zip)
[lnf_latest.pt.zip](https://github.com/user-attachments/files/22430697/lnf_latest.pt.zip)
[m4-sWE-0.1B.script.pt.zip](https://github.com/user-attachments/files/17967188/m4-sWE-0.1B.script.pt.zip)
[mask_depthwise_conv.pt.zip](https://github.com/lutzroeder/netron/files/6186157/mask_depthwise_conv.pt.zip)
[mask_model.pt.zip](https://github.com/lutzroeder/netron/files/10080302/mask_model.pt.zip)
[mask_rcnn.pt.zip](https://github.com/user-attachments/files/17966950/mask_rcnn.pt.zip)
[mnist_linear_torchscript.zip](https://github.com/lutzroeder/netron/files/3585288/mnist_linear_torchscript.zip)
[mobilefacenet.pth.zip](https://github.com/lutzroeder/netron/files/13594924/mobilefacenet.pth.zip)
[mobilenet_v2.pt.zip](https://github.com/lutzroeder/netron/files/6096624/mobilenet_v2.pt.zip)
[mobilenet_v2_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096623/mobilenet_v2_traced.pt.zip)
[model.ptl.zip](https://github.com/lutzroeder/netron/files/11149538/model.ptl.zip)
[model_dynamic_cpu.pt.zip](https://github.com/user-attachments/files/16091313/model_dynamic_cpu.pt.zip)
[model_static_cpu.pt.zip](https://github.com/user-attachments/files/16091311/model_static_cpu.pt.zip)
[module_000007.pt.zip](https://github.com/lutzroeder/netron/files/9075631/module_000007.pt.zip)
[nlf_l_multi.torchscript.zip](https://github.com/user-attachments/files/20839705/nlf_l_multi.torchscript.zip)
[pedestrian_interaction_position_embedding.pt.zip](https://github.com/user-attachments/files/16120032/pedestrian_interaction_position_embedding.pt.zip)
[pedestrian_interaction_single_lstm.pt.zip](https://github.com/user-attachments/files/16120033/pedestrian_interaction_single_lstm.pt.zip)
[pyg_model.pt.zip](https://github.com/user-attachments/files/17969647/pyg_model.pt.zip)
[r3d_18.pt.zip](https://github.com/lutzroeder/netron/files/6096650/r3d_18.pt.zip)
[r3d_18_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096649/r3d_18_traced.pt.zip)
[rcnn.pt.zip](https://github.com/lutzroeder/netron/files/9035740/rcnn.pt.zip)
[refine_model.pt.zip](https://github.com/lutzroeder/netron/files/6186158/refine_model.pt.zip)
[resnet101.pt.zip](https://github.com/lutzroeder/netron/files/6096667/resnet101.pt.zip)
[resnet101_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096666/resnet101_traced.pt.zip)
[resnet18_quantized_cifar10.pt.zip](https://github.com/lutzroeder/netron/files/7610907/resnet18_quantized_cifar10.pt.zip)
[resnet50_pertensor.zip](https://github.com/lutzroeder/netron/files/5212006/resnet50_pertensor.zip)
[resnext50_32x4d_fpn.pth.zip](https://github.com/lutzroeder/netron/files/8985800/resnext50_32x4d_fpn.pth.zip)
[rpn_model.pt.zip](https://github.com/lutzroeder/netron/files/6186159/rpn_model.pt.zip)
[shufflenet_v2_x1_0.pt.zip](https://github.com/lutzroeder/netron/files/6096671/shufflenet_v2_x1_0.pt.zip)
[shufflenet_v2_x1_0_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096670/shufflenet_v2_x1_0_traced.pt.zip)
[squeezenet1_1.pt.zip](https://github.com/lutzroeder/netron/files/6096662/squeezenet1_1.pt.zip)
[squeezenet1_1_traced.pt.zip](https://github.com/lutzroeder/netron/files/6096661/squeezenet1_1_traced.pt.zip)
[squeezenet1_1_trt.pt.zip](https://github.com/user-attachments/files/18792413/squeezenet1_1_trt.pt.zip)
[ssdlite320_mobilenet_v3_large.pt.zip](https://github.com/lutzroeder/netron/files/7677468/ssdlite320_mobilenet_v3_large.pt.zip)
[stable_diffusion.zip](https://github.com/lutzroeder/netron/files/10199037/stable_diffusion.zip)
[test.8bit.pth.zip](https://github.com/lutzroeder/netron/files/5238524/test.8bit.pth.zip)
[TestSerialization.test_lstm.traced.pt.zip](https://github.com/user-attachments/files/16121906/TestSerialization.test_lstm.traced.pt.zip)
[TFModel_traced_eager_quant.pt.zip](https://github.com/lutzroeder/netron/files/10867120/TFModel_traced_eager_quant.pt.zip)
[torchscript_resnet50_fp32.pth.zip](https://github.com/lutzroeder/netron/files/7688572/torchscript_resnet50_fp32.pth.zip)
[traced_fft.zip](https://github.com/lutzroeder/netron/files/5172197/traced_fft.zip)
[traced_gpt2.pt.zip](https://github.com/lutzroeder/netron/files/12010345/traced_gpt2.pt.zip)
[traced_pseudo_quantized_model.pt.zip](https://github.com/user-attachments/files/19859244/traced_pseudo_quantized_model.pt.zip)
[transformer.pt.zip](https://github.com/lutzroeder/netron/files/10271969/transformer.pt.zip)
[transformer_traced.pt.zip](https://github.com/lutzroeder/netron/files/10271968/transformer_traced.pt.zip)
[tuple_repro_dynidx.pt.zip](https://github.com/user-attachments/files/27087592/tuple_repro_dynidx.pt.zip)
[unknown_type_name.pt.zip](https://github.com/user-attachments/files/21330384/unknown_type_name.pt.zip)
[v1_lj_8000.jit.zip](https://github.com/user-attachments/files/16041474/v1_lj_8000.jit.zip)
[wav2mel.pt.zip](https://github.com/lutzroeder/netron/files/7534717/wav2mel.pt.zip)
[yolo4_tiny.pt.zip](https://github.com/lutzroeder/netron/files/7995416/yolo4_tiny.pt.zip)
[yolox_m.torchscript.pt.zip](https://github.com/lutzroeder/netron/files/15031984/yolox_m.torchscript.pt.zip)

**Tools:**
[torch_jit_debug.py.zip](https://github.com/user-attachments/files/18427483/torch_jit_debug.py.zip)
[torch_jit_schemas.py.zip](https://github.com/user-attachments/files/18466194/torch_jit_schemas.py.zip)

**Tasks:**
- [ ] `prim::Loop`
- [ ] `prim::GetAttr`
- [ ] `_jit_pass_inline`
- [ ] Implicit inputs
- [ ] Nested graphs
